### PR TITLE
[wip] Setup/fix version

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

File: testing/runtests_travis.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import os
+
+if os.environ['TOXENV']:
+    import tox
+    tox.cmdline()
+elif os.environ['SELFINSTALL']:
+    call('python setup.py sdist', shell=True)
+    call('easy_install dist/*', shell=True)
+    import pkg_ressources
```
Comment (jaraco) [inline testing/runtests_travis.py]: This looks misspelled.

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ python:
 - '3.6-dev'
 env:
 - TOXENV=py-test
+- SELFINSTALL=1
```
Comment (jaraco) [inline .travis.yml]: Maybe consider instead just having two TOXENVs, one which is py-test and the other self-install... and perform the self-install checks in that toxenv.

File: testing/runtests_travis.py (hunk 1 above)
Comment (RonnyPfannschmidt) [reply testing/runtests_travis.py]: thansk for reviwing, i got sidetracked, i hope to finish until the weekend

File: .travis.yml (hunk 2 above)
Comment (RonnyPfannschmidt) [reply .travis.yml]: the correctly created sdist of tox is a problem, thus i choose to make different travis level calls

**Pull request merged**